src/SDKs/Azure.ApplicationModel.Configuration/data-plane/Azure.ApplicationModel.Configuration/SettingSelector.cs

```diff
@@ -81,7 +82,16 @@ public override bool Equals(object obj)
         }
 
         [EditorBrowsable(EditorBrowsableState.Never)]
-        public override int GetHashCode() => base.GetHashCode();
+        public override int GetHashCode()
+        {
+            var hashCode = new HashCodeBuilder();
+            hashCode.Add(Keys);
+            hashCode.Add(Labels);
+            hashCode.Add(AsOf);
+            hashCode.Add(Fields);
+            hashCode.Add(BatchLink, StringComparer.InvariantCultureIgnoreCase);
```

Comment:
Are you sure? Equals uses `StringComparison.Ordinal`  for BatchLink.

Reply:
Same for ConfigurationSetting.cs

Reply:
My bad :(. 
Fixed

Reply:
Is there a reason we use `StringComparer.Ordinal`?

Reply:
Talked about this offline.